src/backlight/backlight.c

```diff
@@ -13,11 +13,37 @@
 #include "main-func.h"
 #include "mkdir.h"
 #include "parse-util.h"
+#include "pretty-print.h"
+#include "terminal-util.h"
 #include "reboot-util.h"
 #include "string-util.h"
 #include "strv.h"
 #include "util.h"
 
+static int help(void) {
+        _cleanup_free_ char *link = NULL;
+        int r;
+
+        r = terminal_urlify_man("systemd-backlight", "8", &link);
+        if (r < 0)
+                return log_oom();
+
+        printf("%s [save|load] [backlight|leds]:[DEVICE]\n"
+               "\n%sSave and resume backlight brightness at shutdown and boot.%s\n\n"
+               "  save            Save current brightness\n"
+               "  load            Set brightness to be the previously saved value\n"
+               "  backlight       Operate backlight brightness\n"
+               "  leds            Operate LED brightness\n"
+               "  DEVICE          Device to be operated\n"
```

Comment:
we explain verbs here only typically, i.e. these final three lines should go as people would assume they were supported verbs, but they aren't.

Reply:
removed final three